mobile_app_connector/static/mappings/wp_post_mapping.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "mobile_app_wp_post",
+  "model": "wp.post",
+  "mapping": {
+    "Blog": {
+      "sub_mapping": {
+        "name": "mobile_app_wp_post_blog",
+        "model": "wp.post",
+        "mapping": {
+          "ImageUrl": {
+            "field": "image_url"
+          },
+          "Title": {
+            "field": "name"
+          },
+          "Url": {
+            "field": "url"
+          },
+          "WP_id": {
+            "field": "wp_id"
+          },
+          "Post_type": {
+            "field": "post_type"
+          }
+        }
+      }
+    },
+    "Title": {
+      "field": "name"
+    },
+    "ActionText": {
+      "field": "image_url"
+    },
```

Comment:
The posts in the app have the image_url in the tile title

Reply:
![Screenshot_20200221-115329](https://user-images.githubusercontent.com/17401810/75033240-c71d9680-54aa-11ea-995c-28b0ac52ecb8.jpg)